test/primitives/array-element-test.js

```diff
@@ -195,6 +236,15 @@ describe('ArrayElement', function() {
     //     expect(items).to.have.length(4);
     //   });
     // });
+
+    describe('#clone', function() {
+      it('creates a deep clone of the element', function() {
+        var clone = arrayElement.clone();
+        expect(clone).to.be.instanceOf(minim.ArrayElement);
+        expect(clone).to.not.equal(arrayElement);
```

Comment:
Ah, well looking again you convert it for this to work. So is there no way for an element to == another but not deep equal (i.e. be the same instance)?